src/app/submission/sections/duplicates/section-duplicates.component.spec.ts

```diff
@@ -0,0 +1,248 @@
+import { getMockFormService } from '../../../shared/mocks/form-service.mock';
+import { FormService } from '../../../shared/form/form.service';
+import { SubmissionFormsConfigDataService } from '../../../core/config/submission-forms-config-data.service';
+import { SectionsType } from '../sections-type';
+import { mockSubmissionCollectionId, mockSubmissionId } from '../../../shared/mocks/submission.mock';
+import { JsonPatchOperationPathCombiner } from '../../../core/json-patch/builder/json-patch-operation-path-combiner';
+import { SubmissionSectionDuplicatesComponent } from './section-duplicates.component';
+import { CollectionDataService } from '../../../core/data/collection-data.service';
+import { JsonPatchOperationsBuilder } from '../../../core/json-patch/builder/json-patch-operations-builder';
+import { SectionFormOperationsService } from '../form/section-form-operations.service';
+import { SubmissionScopeType } from '../../../core/submission/submission-scope-type';
+import { License } from '../../../core/shared/license.model';
+import { Collection } from '../../../core/shared/collection.model';
+import { ObjNgFor } from '../../../shared/utils/object-ngfor.pipe';
+import { VarDirective } from '../../../shared/utils/var.directive';
+import { PaginationService } from '../../../core/pagination/pagination.service';
+import { PaginationServiceStub } from '../../../shared/testing/pagination-service.stub';
+import { Duplicate} from '../../../shared/object-list/duplicate-data/duplicate.model';
+import { MetadataValue } from '../../../core/shared/metadata.models';
+import { defaultUUID } from '../../../shared/mocks/uuid.service.mock';
+import { DUPLICATE } from '../../../shared/object-list/duplicate-data/duplicate.resource-type';
+
+function getMockSubmissionFormsConfigService(): SubmissionFormsConfigDataService {
+  return jasmine.createSpyObj('FormOperationsService', {
+    getConfigAll: jasmine.createSpy('getConfigAll'),
+    getConfigByHref: jasmine.createSpy('getConfigByHref'),
+    getConfigByName: jasmine.createSpy('getConfigByName'),
+    getConfigBySearch: jasmine.createSpy('getConfigBySearch')
+  });
+}
+
+function getMockCollectionDataService(): CollectionDataService {
+  return jasmine.createSpyObj('CollectionDataService', {
+    findById: jasmine.createSpy('findById'),
+    findByHref: jasmine.createSpy('findByHref')
+  });
+}
+
+const duplicates: Duplicate[] = [{
+    title: 'Unique title',
+    uuid: defaultUUID,
+    workflowItemId: 1,
+    workspaceItemId: 2,
+    owningCollection: 'Test Collection',
+    metadata: {
+      'dc.title': [
+        Object.assign(new MetadataValue(), {
+        'value': 'Unique title',
+        'language': null,
+        'authority': null,
+        'confidence': -1,
+        'place': 0
+      })]
+    },
+    type: DUPLICATE,
+    _links: {
+      self: {
+        href: 'http://localhost:8080/server/api/core/items/search/findDuplicates?uuid=testid'
```

Comment:
URL needs updating to match new endpoint

Reply:
done